src/main/kotlin/net/atos/zac/identity/IdentityService.kt

```diff
@@ -60,4 +67,22 @@ class IdentityService @Inject constructor(
             .map { it.toUser() }
             .sortedBy { it.getFullName() }
     }
+
+    fun checkIfUserIsInGroup(userId: String?, groupId: String?) {
```

Comment:
Why are these two arguments optional? Does not make any sense I think? Why would you call this function without a user or group ID?

Reply:
Reworked this. Now the null check is part of `ZaakRestService`